crates/committed/src/report.rs

```diff
@@ -63,7 +63,7 @@ pub(crate) enum Content<'s> {
     NoPunctuation(NoPunctuation),
     Imperative(Imperative<'s>),
     Wip(Wip),
-    Fixup(Fixup),
+    Autosquash(Autosquash),
```

Comment:
This change is unrelated to this commit and *is* a breaking change as discussed on the other PRs

Reply:
To be clear, my expectation is this commit will continue to emit `Fixup`